### PR TITLE
feat(reactions): add sendReaction on meeting plugin

### PR DESCRIPTION
**Pull request by torgeadelin**
# COMPLETES [#SPARK-393047](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-393047)

## This pull request addresses

Add new public method `sendReaction` on meetings plugin on the meeting object.

## by making the following changes

- add new request under plugin-meetings/meeting/request
- add new public method on the meeting object

### Change Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Tooling change
- [ ] Internal code refactor

## The following scenarios where tested

Unit tests, manual testing.

### I certified that

- [x] I have read and followed [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request)
- [x] I discussed changes with code owners prior to submitting this pull request

- [x] I have not skipped any automated checks
- [x] All existing and new tests passed
- [ ] I have updated the documentation accordingly

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

File: packages/@webex/plugin-meetings/src/meeting/request.ts [hunk 1]
```diff
@@ -768,4 +769,26 @@ export default class MeetingRequest extends StatelessWebexPlugin {
...
+  /**
+   * Make a network request to send a reaction.
+   * @param {Object} options
+   * @param {Url} options.reactionChannelUrl
+   * @param {Reaction} options.reaction
+   * @param {string} options.senderID
+   * @returns {Promise}
+   */
+  sendReaction({ reactionChannelUrl, reaction, participantId }: { reactionChannelUrl: string, reaction: Reaction, participantId: string }) {
+    const uri = `${reactionChannelUrl}`;
```
Comment (InteractiveTimmy) [inline packages/@webex/plugin-meetings/src/meeting/request.ts:782]: Is there a reason we're using a template literal here?

**Review by InteractiveTimmy — APPROVED** [review]
LGTM, question around the template literal, non-blocking.

File: packages/@webex/plugin-meetings/src/meeting/index.ts [hunk 2]
```diff
@@ -6212,4 +6213,35 @@ export default class Meeting extends StatelessWebexPlugin {
...
+    const reaction: Reaction = {
+      ...reactionData,
+      tone: skinToneData
+    };
+
+    if (reactionChannelUrl) {
+      return this.meetingRequest.sendReaction({reactionChannelUrl, reaction, participantId});
+    }
+
+    return Promise.reject(new Error('Reaction Channel URL not found.'));
```
Comment (arun3528) [inline packages/@webex/plugin-meetings/src/meeting/index.ts:6245]: can we change the message to more developer friendly errors , as they dont understand what a reaction channel url 

may be add something like "Error sending Reaction, service url not found"

**Comment by arun3528** [comment]
Other then the name change looks good to me

File: packages/@webex/plugin-meetings/src/meeting/request.ts (hunk 1 above)
Comment (torgeadelin) [reply packages/@webex/plugin-meetings/src/meeting/request.ts:782]: probably used one of the existing functions as a starting point and didn't remove the template literal (which was useful in the other functions). Will remove.

File: packages/@webex/plugin-meetings/src/meeting/index.ts (hunk 2 above)
Comment (torgeadelin) [reply packages/@webex/plugin-meetings/src/meeting/index.ts]: addressed

File: packages/@webex/plugin-meetings/src/meeting/request.ts (hunk 1 above)
Comment (torgeadelin) [reply packages/@webex/plugin-meetings/src/meeting/request.ts]: Removed.

**Pull request merged**